roles/nova-common/tasks/main.yml

```diff
@@ -14,11 +14,17 @@
 - file: dest=/var/log/nova state=directory mode=0755 owner=nova group=nova
 
 - name: nova config
-  action: template src={{ item }} dest=/etc/nova mode=0644
+  action: template src={{ item }} dest=/etc/nova mode=0640
+          owner=nova group=nova
   with_fileglob: ../templates/etc/nova/*
   notify:
     - restart nova services
 
+- name: set nova log file permissions
+  file: dest="{{ item.paths[0] }}" mode=0640 state=touch
+        owner=nova group=nova
+  with_items: nova.logs
+
```

Comment:
The problem is that we are enforcing log management and who owns the logs via a component consuming them (ansible and logstash-forwarder's config data), and not the component which produces the logs in the first place.

I do not immediately see a way for nova to configure who it writes the logs as, and we might be able to solve that in another method but this is not the correct approach unless we expect to run ansible continuously, which will not happen.

Remember, ansible executs on command whereas Chef runs "continuously" in the background. Though this is likely less than ideal in either tool, Chef would have supported this reasonably well by its execution behavior.